webdriver/tests/bidi/__init__.py

```diff
@@ -86,11 +86,11 @@ async def get_device_pixel_ratio(bidi_session, context):
 
     :param bidi_session: BiDiSession
     :param context: Browsing context ID
-    :returns: (int) devicePixelRatio.
+    :returns: (float) devicePixelRatio.
```

Comment:
I don't want to change it in this PR as I am not certain that it would work everywhere. Do you want to open a follow-up? I bet there are other places where a similar change can be applied?

Reply:
OK sure. I'll add it to my ongoing PR that deals with similar changes then.